index.d.ts

```diff
@@ -472,14 +486,14 @@ console.log(stdout);
 const subprocess = execa('node');
 
 setTimeout(() => {
-	subprocess.cancel()
+  subprocess.cancel()
```

Comment:
Don't make unrelated changes. It should use tab indentation.

Reply:
Fix in latest commit.